addons/base_vat/models/res_partner.py

```diff
@@ -33,6 +33,7 @@
     'br': _('either 11 digits for CPF or 14 digits for CNPJ'),
     'ch': _('CHE-123.456.788 TVA or CHE-123.456.788 MWST or CHE-123.456.788 IVA'),  # Swiss by Yannick Vaucher @ Camptocamp
     'cl': 'CL76086428-5',
+    'cr': _('CR123456789 or 123456789'),
```

Comment:
alphabetic order please
also please use valid VAT numbers as a reference, and I think only one should suffice, since they're the same